x/stakeibc/keeper/unbonding_records_get_host_zone_unbondings_msgs_test.go

```diff
@@ -132,6 +132,26 @@ func (s *KeeperTestSuite) CheckUnbondingMessages(tc UnbondingTestCase, expectedU
 		s.Require().Equal(expected.UnbondAmount.Int64(), actualSplit.Amount.Int64(), "callback message amount - index %d", i)
 	}
 
+	// Check the delegation change in progress was incremented from each that had an unbonding
+	actualHostZone, found := s.App.StakeibcKeeper.GetHostZone(s.Ctx, HostChainId)
+	s.Require().True(found, "host zone should have been found")
+
+	for _, actualValidator := range actualHostZone.Validators {
+		validatorUnbonded := false
+		for _, unbondedVal := range expectedUnbondings {
+			if actualValidator.Address == unbondedVal.Validator {
```

Comment:
This is right given the types in the unbondings, the naming bugs me because it looks like comparing an Address to a Validator struct

Reply:
do you have a suggestion or just noting it?

Reply:
The only way to make it clearer to a reader would be to use an extra variable with the word address in it so it is clear its an address compared to address like:  expectedValidatorAddress := unbondedVal.Validator   but probably not work it, I will remove the comment